homeassistant/components/rest/data.py

```diff
@@ -56,6 +56,15 @@ def __init__(
         self.last_exception: Exception | None = None
         self.headers: httpx.Headers | None = None
 
+    @property
+    def payload(self) -> str | None:
+        """Get request data."""
+        return self._request_data
```

Comment:
This property is not needed, `_request_data` is accessed directly by `async_update`.